compiler/runtime.go

```diff
@@ -95,14 +99,19 @@ func (e *blockEvaluator) Type() semantic.MonoType {
 
 func (e *blockEvaluator) Eval(ctx context.Context, scope Scope) (values.Value, error) {
 	var err error
-	var value values.Value
-	for _, b := range e.body {
+	var v values.Value
```

Comment:
I was more thinking to do the following:

```go
for _, b := range e.body[:len(e.body)-1] {
    value, err := eval(ctx, b, scope)
    if err != nil {
        return nil, err
    }
    value.Release()
}
return eval(ctx, e.body[len(e.body)-1], scope)
```

I believe it is an invariant the body will have at least one statement. If that's the case, this code should be safe.